### DISCUSSION TITLE
Command to jump to last pane(buffer)

### DISCUSSION THREAD

**Discussion by smpallen99** [zed-industries/zed#7535] [Feature Requests]
I use the `:b#` command in vim hundred's of times a day. I have it mapped to <leader> r. It jumps to the last active buffer. It is very useful for several secenarions.

* jumping between code and test files
* open a buffer, then return to your previous buffer

Is there a way to do this in Zed currently? If not, any plans for the feature.

BTW, this one of the most import features holding me back from switch from neovide (neovim) full time.

**Comment by alygin** [comment]
As soon as #6947 makes it to a release, `cmd+p enter` will do the job.

**Reply by smpallen99** [reply]
That will work prefectly. One thing left...Map it to my leader key

**Reply by alygin** [reply]
Ah, I'm afraid you won't be able to map it to a single keystroke. Yet. At the moment, `cmd+p` opens the file finder panel with the previously opened file selected automatically, and `enter` confirms the selection.

[This feature](https://github.com/zed-industries/zed/discussions/8310), if implemented, may simplify shortcut to something repeated twice (like `cmd+p cmd+p`). Then, with the [tab switcher](https://github.com/zed-industries/zed/pull/7987#issuecomment-1951717141) implemented, it'll become a single keystroke (probably `ctrl+tab` by default).

**Comment by cseelus** [comment]
I use `,` (comma) as my leader key in VIM since ages.

So I mapped pressing it twice to opening the tab switcher and pressing enter, which switches to your last used tab/buffer.

keymap.json:
`", ,": ["workspace::SendKeystrokes", "ctrl-tab enter"]`

If you don't want to manually define new key bindings, you could also just press cmd-p-p, which achieves the same thing.
